tests/kafkatest/benchmarks/core/benchmark_test.py

```diff
@@ -36,47 +35,42 @@ class Benchmark(Test):
     """
     def __init__(self, test_context):
         super(Benchmark, self).__init__(test_context)
-        self.num_zk = 1
         self.num_brokers = 3
         self.topics = {
             TOPIC_REP_ONE: {'partitions': 6, 'replication-factor': 1},
             TOPIC_REP_THREE: {'partitions': 6, 'replication-factor': 3}
         }
 
-        self.zk = ZookeeperService(test_context, self.num_zk)
-
         self.msgs_large = 10000000
         self.batch_size = 8*1024
         self.buffer_memory = 64*1024*1024
         self.msg_sizes = [10, 100, 1000, 10000, 100000]
         self.target_data_size = 128*1024*1024
         self.target_data_size_gb = self.target_data_size/float(1024*1024*1024)
 
-    def setUp(self):
-        self.zk.start()
-
     def start_kafka(self, security_protocol, interbroker_security_protocol, version, tls_version=None):
         self.kafka = KafkaService(
             self.test_context, self.num_brokers,
-            self.zk, security_protocol=security_protocol,
+            zk=None, security_protocol=security_protocol,
             interbroker_security_protocol=interbroker_security_protocol, topics=self.topics,
             version=version, tls_version=tls_version)
         self.kafka.log_level = "INFO"  # We don't DEBUG logging here
         self.kafka.start()
 
-    @cluster(num_nodes=5)
-    @parametrize(acks=1, topic=TOPIC_REP_ONE)
-    @parametrize(acks=1, topic=TOPIC_REP_THREE)
-    @parametrize(acks=-1, topic=TOPIC_REP_THREE)
-    @matrix(acks=[1], topic=[TOPIC_REP_THREE], message_size=[10, 100, 1000, 10000, 100000], compression_type=["none", "snappy"], security_protocol=['SSL'], tls_version=['TLSv1.2', 'TLSv1.3'])
-    @matrix(acks=[1], topic=[TOPIC_REP_THREE], message_size=[10, 100, 1000, 10000, 100000], compression_type=["none", "snappy"], security_protocol=['PLAINTEXT'])
-    @cluster(num_nodes=7)
-    @parametrize(acks=1, topic=TOPIC_REP_THREE, num_producers=3)
+    @cluster(num_nodes=9)
+    @parametrize(acks=1, topic=TOPIC_REP_ONE, metadata_quorum=quorum.isolated_kraft)
+    @parametrize(acks=1, topic=TOPIC_REP_THREE, metadata_quorum=quorum.isolated_kraft)
+    @parametrize(acks=-1, topic=TOPIC_REP_THREE, metadata_quorum=quorum.isolated_kraft)
+    @matrix(acks=[1], topic=[TOPIC_REP_THREE], message_size=[10, 100, 1000, 10000, 100000],
+            compression_type=["none", "snappy"], security_protocol=['SSL'], tls_version=['TLSv1.2', 'TLSv1.3'], metadata_quorum=quorum.all_kraft)
+    @matrix(acks=[1], topic=[TOPIC_REP_THREE], message_size=[10, 100, 1000, 10000, 100000],
+            compression_type=["none", "snappy"], security_protocol=['PLAINTEXT'], metadata_quorum=quorum.all_kraft)
+    @parametrize(acks=1, topic=TOPIC_REP_THREE, num_producers=3, metadata_quorum=quorum.isolated_kraft)
```

Comment:
What's the reason for using `quorum.isolated_kraft` for some tests and `quorum.all_kraft` for the others? In my opinion, we could use `quorum.isolated_kraft` for all.

Reply:
Originally, I followed other test case to use `isolated_kraft` for `parametrize` and `all_kraft` for `matrix`, but I think we don't need to test both `isolated_kraft` and `combined_kraft` here. Updated it. Thanks.

https://github.com/apache/kafka/blob/f6f41dc5ebc0933210c90a520258f89900d67857/tests/kafkatest/sanity_checks/test_verifiable_producer.py#L124

https://github.com/apache/kafka/blob/f6f41dc5ebc0933210c90a520258f89900d67857/tests/kafkatest/sanity_checks/test_verifiable_producer.py#L43-L47